src/FailureStrategy/BehaviorRemovingStrategy.php

```diff
@@ -0,0 +1,83 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yiisoft\Yii\Queue\FailureStrategy;
+
+use Yiisoft\Yii\Queue\Message\MessageInterface;
+use Yiisoft\Yii\Queue\Payload\PayloadInterface;
+
+final class BehaviorRemovingStrategy implements FailureStrategyInterface
```

Comment:
What is the use case for it?

Reply:
E.g. to remove delay from the failed message before redirect it to the same or to another queue.

Reply:
Should it be named "remove delay" or sometihing?

Reply:
It is just an example. It can remote any behavior declared in message metadata.

Reply:
@samdark, as of #68 this strategy will be more straightforward.